02-express-tutorial/data.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Unnecessary space here. I believe this data was provided, but since you added this it makes it show up in the PR which is making more of your code to review.